packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -192,8 +192,9 @@ export interface FieldProps<TCustomMetadata = unknown> {
 	 * Optional metadata to associate with the field.
 	 *
 	 * @remarks
-	 * Note: this metadata is not persisted; it is strictly local to the application.
-	 * So, making changes to these values will not affect collaborators.
+	 * Note: this metadata is not persisted nor made part of the collaborative state; it is strictly client-local.
+	 * Different clients in the same collaborative session may see different metadata for the same field.
+	 * For the same reason, an application may change the properties of this metadata without fear of breaking collaboration.
```

Comment:
Just a bit wary of this sentence... if the application decides to behave differently depending on the metadata it sees (e.g. updates collaborative state based on which nodes were deemed searchable), two clients in the same session but with different metadata could lead to not _technically_ breaking collaboration, but yes for app purposes? Instead of this I would've thought of writing something like "applications should be careful to handle the case where different clients see different metadata in a way that makes sense for them".